pytorch_lightning/trainer/connectors/env_vars_connector.py

```diff
@@ -12,29 +12,12 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from functools import wraps
-from typing import Callable
+from pytorch_lightning.utilities import rank_zero_deprecation
 
-from pytorch_lightning.utilities.argparse import get_init_arguments_and_types, parse_env_variables
+rank_zero_deprecation(
```

Comment:
why do we need this deprecation if the function is protected?

Reply:
https://github.com/PyTorchLightning/pytorch-lightning/issues/10161#issuecomment-953647498

Reply:
What I understand from @tchaton's comment is that the functionality needs to stay.
As in, the ability to pass from the environment.

Not that users are actually importing this function

Reply:
I agree, I think you can just move the code to the utilities file, no need for deprecation warning. thanks for fixing this!

Reply:
@kaushikb11 Let's drop the `rank_zero_deprecation`

Reply:
Updated!